docs/specification/1.0.md

```diff
@@ -126,7 +128,7 @@ Field | Description
 `hook` |*string* or *URL*. The hook that triggered this CDS Service call<br />(todo: link to hook documentation)
 <nobr>`hookInstance`</nobr> |*string*.  A UUID for this particular hook call (see more information below)
 `fhirServer` |*URL*.  The base URL EHR's [FHIR](https://www.hl7.org/fhir/) server. The scheme should be `https`
-`fhirAuthorization` | *object*. The OAuth 2 authorization providing access to the EHR's FHIR server. See the [FHIR Resource Access](#fhir-resource-access) heading of the security section for more information.
+`fhirAuthorization` | *object*. The OAuth 2.0 bearer access token granting the CDS Service access to FHIR resources. See the [FHIR Resource Access](#fhir-resource-access) heading of the security section for more information.
```

Comment:
The `fhirAuthorization` field isn't the bearer token. It is an object that contains the bearer token along with supplemental information related to said token.

Reply:
@kpshek.  Agree that fhirAuthorization contains more than the access token.   I think the most accurate description is to describe it as an "authorization structure."